core/src/main/java/com/sequenceiq/cloudbreak/service/workspace/WorkspaceService.java

```diff
@@ -225,25 +226,39 @@ public Workspace getByNameForUserOrThrowNotFound(String orgName, User currentUse
         }
     }
 
-    public Workspace deleteByNameForUser(String orgName, User currentUser, Workspace defaultWorkspace) {
+    public Workspace deleteByNameForUser(String workspaceName, User currentUser, Workspace defaultWorkspace) {
         try {
             return transactionService.required(() -> {
-                Workspace workspaceForDelete = getByNameForUserOrThrowNotFound(orgName, currentUser);
+                Workspace workspaceForDelete = getByNameForUserOrThrowNotFound(workspaceName, currentUser);
                 authorizeWorkspaceManipulation(currentUser, workspaceForDelete, Action.MANAGE, "You cannot delete this workspace.");
 
                 workspaceModificationVerifierService.checkThatWorkspaceIsDeletable(currentUser, workspaceForDelete, defaultWorkspace);
                 Long deleted = userWorkspacePermissionsService.deleteByWorkspace(workspaceForDelete);
                 setupDeletionDateAndFlag(workspaceForDelete);
                 workspaceRepository.save(workspaceForDelete);
-                LOGGER.info("Deleted organisation: {}, related permissions: {}", orgName, deleted);
+                LOGGER.info("Deleted workspace: {}, related permissions: {}", workspaceName, deleted);
                 return workspaceForDelete;
             });
         } catch (TransactionExecutionException e) {
             throw new TransactionRuntimeExecutionException(e);
         }
     }
 
-    private Map<User, Set<String>> orgUserPermissionJsonSetToMap(Set<ChangeWorkspaceUsersJson> updateWorkspaceUsersJsons) {
+    private void checkRemovable(String workspaceName, Set<String> userIds, User currentUser)  {
```

Comment:
ensureWorkspaceManagement would be a better name. checkRemovable is not very self-explanatory

Reply:
because it should only work for user removes maybe renaming is not enough